src/blocks/CallToAction/index.module.scss

```diff
@@ -1,43 +1,54 @@
 @use '@scss/common.scss' as *;
 
-.callToAction {
+.wrapper {
   position: relative;
 }
 
-.bgWrapper {
-    width: 100%;
-    height: 100%;
-    position: absolute;
-    top: 0;
-    left: 0;
-}
+.container {
+  position: relative;
+  padding-top: calc(var(--base) * 10);
+  padding-bottom: calc(var(--base) * 10);
 
-.bgGutter {
-    width: 100%;
-    height: 100%;
+  @include mid-break {
+    row-gap: calc(var(--base) * 3);
+  }
 }
 
-.bg1 {
-    position: relative;
-    height: 100%;
-    margin-left: calc(var(--gutter-h) / -2);
-    margin-right: calc(var(--gutter-h) / -2);
-    width: calc(100% + var(--gutter-h));
-    background-color: var(--color-base-950);
+.linksContainer {
+  position: relative;
+
+  :global(.crosshair) {
+    display: block;
 
     @include mid-break {
-        margin: 0;
-        width: 100%;
+      display: none;
     }
+  }
 }
 
-.contentWrap {
-    position: relative;
-    padding: calc(var(--base) * 4) 0;
+.scanline {
+  top: -20%;
+  height: 140%;
```

Comment:
Try `top: -25%` & `height: 150%;`

This to me looks more inline with the design